helm/values.yaml

```diff
@@ -63,19 +63,18 @@ microservice-chart:
     idpay-eventhub-00:
       KAFKA_BINDER_BROKER_ENDPOINT_LIST: kafka_broker
       KAFKA_TOPIC_NOTIFICATION: idpay_notification_request_topic
+
```

Comment:
remove also KAFKA_TOPIC_NOTIFICATION configuration: it should not more used (maybe you could remove the entire idpay-eventhub-00 block...)